Answers.md

```diff
@@ -0,0 +1,16 @@
+### Describe the following: DataBase, Collection , Document
+    Database: In Mongo a database is where we save the collections.
+    Collection: In Mongo a colection is where we store our documents.
+    Document: Basically is a schema that represents the information format in our collention.
+    
+
+### Describe how one can achieve the pattern of relationships in MongoDB. What needs to take place at the schema level? How do we 'fill' in the appropriate relational data using mongoose?
+    We need to determine which kind of relationship we want and after that we have to add the Objectid of the referenced
+    collection into the target collection.
```

Comment:
To answer the "fill" in question, you would use populate along with the query.

Reply:
I rephrased in a more eloquent way  I hope 👍